Console5/Console5/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите слово или фразу на русском языке");
+        Console.WriteLine("--------------------------------------");
+        string leet = Console.ReadLine().ToLower();
+        Console.WriteLine(TranslateOnLeet(leet));
+        Console.ReadKey();
+    }
+    static string TranslateOnLeet(string x)
```

Comment:
Названия переменной и метода перекочевали из другого варианта. Тут они неуместны, более того, могут ввеси в заблуждение при чтении кода.
```suggestion
        string text = Console.ReadLine().ToLower();
        Console.WriteLine(Transliterate(text));
        Console.ReadKey();
    }
    static string Transliterate(string x)
```